packages/create/templates/hello-prisma/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
This shouldn't be here. Add .env to the main gitignore and also the prisma template, if it isn't there already.

Reply:
Done.